client/components/App.jsx

```diff
@@ -12,18 +15,16 @@ class App extends React.Component {
 
   render() {
     return (
-      <>
-        <Header />
-        <Filters />
-        <ReviewList />
-        <Footer />
-      </>
+      <div id="reviews_container">
+        <div id="reviews_component">
+          <Header />
+          <Filters />
+          <ReviewList />
+          <Footer />
+        </div>
+      </div>
     );
   }
 }
 
-export default App;
-
-{/*
-        <ReviewList />
-         */}
\ No newline at end of file
+export default App;
```

Comment:
Everything looks really good. You might want to get esLint running at some point, just because I see some small things it will yell at you about...  (like it will want a new line at the end of every .js file)

But the code looks good, and keep on keeping on!!

Reply:
Thank you! And hmm... I thought I am running .eslint, but Phonxay mentioned the same thing. I'll check to see what's going on. Maybe I've disabled some of the rules?